plugin.audio.radio.roks/addon.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="plugin.audio.radio.roks" name="Radio Roks online" version="0.1.0" provider-name="Eugene Kuzin">
+    <requires>
+        <import addon="xbmc.python" version="2.20.0" />
```

Comment:
python version should be 2.24.0 to match jarvis. See: http://kodi.wiki/view/addon.xml